docs/troubleshooting/docker-jcmd.rst

```diff
@@ -1,27 +1,34 @@
-============================================================
-Troubleshooting the ``crate`` Docker container with ``jcmd``
-============================================================
+.. _jcmd-docker:
+
+====================================
+Troubleshooting Docker with ``jcmd``
```

Comment:
This would be now a bit misleading as it's not troubleshooting docker, but it's about troubleshooting CrateDB running in a docker container.

Reply:
@mfussenegger pushed a fixup